argilla-server/src/argilla_server/api/handlers/v1/datasets/records.py

```diff
@@ -428,11 +391,9 @@ async def search_dataset_records(
 async def list_dataset_records_search_suggestions_options(
     *,
     db: AsyncSession = Depends(get_async_db),
-    dataset_id: UUID,
     current_user: User = Security(auth.get_current_user),
+    dataset: Dataset = Depends(get_dataset_or_raise),
 ):
-    dataset = await Dataset.get_or_raise(db, dataset_id)
```

Comment:
It's a lot of boiler plate (the `Depends`) only to get the dataset. Is not easier to left it as it was before?